src/components/dataDisplays/DataDisplay.jsx

```diff
@@ -144,6 +144,7 @@ export default function DataDisplay({
         anchorEl={anchorEl}
         placement="bottom-end"
         transition
+        style={{ zIndex: 9999 }}
```

Comment:
Raising the `zIndex` seems like an acceptable solution. I think that 9999 is too high though. Currently it displays on top of the app header:
<img width="772" alt="overlap" src="https://user-images.githubusercontent.com/50299119/181603819-4290d1bb-460f-49ed-a9c6-bd29edec844b.png">

The app header's z index is `theme.zIndex.drawer + 1`
https://github.com/WildMeOrg/codex-frontend/blob/17a6cb5b0c9e52a97b6e3da7c51f2c8e8419c33c/src/components/AuthenticatedAppHeader/index.js#L60

Maybe for now a zIndex of `theme.zIndex.appBar - 1` would be good enough?

Reply:
It would be interesting to see if we could leverage the stacking context at some point to make sure that the filter popover is always on top within the data display, but otherwise isn't stacking on top of other components because of its z index.

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context